src/common/entity/compute_state_display.ts

```diff
@@ -55,14 +55,21 @@ export const computeStateDisplay = (
     return formatDateTime(date, language);
   }
 
+  let forwardedDomain = domain;
+  if (domain === "group" && stateObj.attributes.device_class) {
+    // Forward the localisation lookup to use binary sensor,
+    // as the device class is forwarded from there, too.
+    forwardedDomain = "binary_sensor";
```

Comment:
A group can be more than just `binary_sensors`, limiting this to `binary_sensor` seems wrong.

Reply:
It's in addition to current behaviour. device_class is entirely optional. I didn't see anything else which can be 'on' or 'off' like a group and had a device_class worth implementing.